build/yaml/crd/old/nsx.vmware.com_securitypolicies.yaml

```diff
@@ -0,0 +1,651 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
crd/old might not be a good name, how about crd/T1/ or crd/legacy

Reply:
ok